packages/webview_flutter/webview_flutter_android/lib/src/android_webview.g.dart

```diff
@@ -1524,18 +1545,45 @@ class WebViewClientHostApi {
       return;
     }
   }
+
+  Future<void> setSynchronousReturnValueForOnRenderProcessGone(
+      int arg_instanceId, bool arg_value) async {
+    final BasicMessageChannel<Object?> channel = BasicMessageChannel<Object?>(
+        'dev.flutter.pigeon.WebViewClientHostApi.setSynchronousReturnValueForOnRenderProcessGone',
+        codec,
+        binaryMessenger: _binaryMessenger);
+    final List<Object?>? replyList = await channel
+        .send(<Object?>[arg_instanceId, arg_value]) as List<Object?>?;
+    if (replyList == null) {
+      throw PlatformException(
+        code: 'channel-error',
+        message: 'Unable to establish connection on channel.',
+      );
+    } else if (replyList.length > 1) {
+      throw PlatformException(
+        code: replyList[0]! as String,
+        message: replyList[1] as String?,
+        details: replyList[2],
+      );
+    } else {
+      return;
+    }
+  }
 }
 
 class _WebViewClientFlutterApiCodec extends StandardMessageCodec {
   const _WebViewClientFlutterApiCodec();
   @override
   void writeValue(WriteBuffer buffer, Object? value) {
-    if (value is WebResourceErrorData) {
+    if (value is RenderProcessGoneDetailData) {
       buffer.putUint8(128);
```

Comment:
Just for my understanding, I noticed the Uint8 value changed from 128 to 129 for WebResourceErrorData and 129 to 130 for WebResourceRequestData. What was the reason RenderProcessGoneDetailData couldn't have been assigned 130 so the other two values could remain the same?

Reply:
Pigeon generation is stateless; it doesn't use previous generator output as an input.